.github/workflows/comments.yml

```diff
@@ -7,60 +7,10 @@ on:
     types:
       - created
 # Remove all permissions by default
-permissions: {}
-# Avoid concurrency over the same issue
-concurrency:
-  group: card-movement-${{ github.event.issue.number }}
+permissions:
+  pull-requests: write
+  issues: write
```

Comment:
We've double-checked that this permissions configuration, and the others in the rest of files, is the correct one.